src/loaders/compute_tools/tool_common.py

```diff
@@ -76,14 +76,13 @@ def __init__(
         -h, --help            show this help message and exit
 
         required named arguments:
+        --env {CI,NEXT,APPDEV,PROD,None}
+                              Environment used to run the tool.
+        --kbase_collection KBASE_COLLECTION
+                              KBase collection identifier name.
         --load_ver LOAD_VER   KBase load version (e.g. r207.kbase.1).
```

Comment:
Hmm, looking at this - the tools are computing on data in `collectionssource/`, right? In which case this isn't a load_ver, it's a source ver.

Reply:
yeah, I see we're using load ver in both the input and output dir, which doesn't really work - we may recompute data from the same source version leading to a new load_ver

Reply:
👍 added `source_ver` and defaults `load_ver` to `source_ver`

Reply:
:+1: